source/content/guides/object-cache-pro/02-installing-configuring.md

```diff
@@ -113,18 +94,14 @@ terminus redis:enable <site>
 	**Terminus:**
 
 	1. Open your terminal.
-	1. Run the following commands:
+	1. Run the following command:
 
 		```bash{promptUser: user}
-		terminus connection:set sftp
 		terminus wp -- <site>.<env> plugin activate object-cache-pro
-		terminus wp -- <site>.<env> redis enable
-		terminus env:commit <site>.<env> --message="Add Object Cache Pro drop-in"
-		terminus connection:set git
```

Comment:
We dropped this from here, but added above. Are we documenting only the "using the scaffold extensions" workflow or not?

Reply:
To my understanding based on our conversations, the plugin will still need to be activated after the `install:run <site>.<env> ocp` is run, but none of these other commands do.